resharper/resharper-unity/src/CSharp/Daemon/UsageChecking/UsageInspectionsSuppressor.cs

```diff
@@ -74,14 +70,15 @@ public bool SuppressUsageInspectionsOnElement(IDeclaredElement element, out Impl
                         return false;
                     }
 
-                    if (IsEventHandler(unityApi, method) || IsRequiredSignatureMethod(method))
+                    if (IsEventHandler(unityApi, method) || IsRequiredSignatureMethod(method) ||
+                        IsImplicitlyUsedInterfaceMethod(method))
```

Comment:
Is it impossible to have implicitly used interface method which is not event function?

Reply:
Yes. I don't think there was any reason to order the `IsImplicitlyUsedInterfaceMethod` before this check.